Makefile

```diff
@@ -15,6 +15,13 @@ clean:
 	git clone https://chromium.googlesource.com/external/gyp.git ./deps/gyp
 	cd deps/gyp && git checkout -q 0bb67471bca068996e15b56738fa4824dfa19de0
 
+# java / jni dependencies
+./deps/java:
+	mkdir -p ./deps/java/test
+	wget http://central.maven.org/maven2/org/hamcrest/hamcrest-core/1.3/hamcrest-core-1.3.jar -O ./deps/java/test/hamcrest-core-1.3.jar
+	wget http://central.maven.org/maven2/junit/junit/4.11/junit-4.11.jar -O ./deps/java/test/junit-4.11.jar
+	wget http://central.maven.org/maven2/com/google/code/findbugs/jsr305/3.0.0/jsr305-3.0.0.jar -O ./deps/java/jsr305-3.0.0.jar
+
```

Comment:
Oh huh, I had thought XCode installed wget, guess not.  Ok, will fix this.  Technically to redistribute binaries, the individual licenses need to be made available some way.  In the past, I've done something like the notices on http://www.yelp.com/static?p=thirdpartyterms (search for e.g. Eigen / libsvm) for stuff released in a mobile client app and the lawyers were ok with citing the licenses this way.  I'll do something similar for this change and (IMO most importantly) cite where the binaries came from.